src/solver/optimisation/opt_fonctions.h

```diff
@@ -60,7 +61,7 @@ bool OPT_PilotageOptimisationQuadratique(PROBLEME_HEBDO*);
 **
 ** \return True si l'operation s'est bien deroulee, false si le probleme n'a pas de solution
 */
-bool OPT_AppelDuSimplexe(PROBLEME_HEBDO*, int, const int);
+bool OPT_AppelDuSimplexe(PROBLEME_HEBDO*, int, const int, std::shared_ptr<optPeriod>);
```

Comment:
Add explanations on this new argument (daily/weekly opt.)

Reply:
Done